libnymea-core/jsonrpc/zigbeehandler.cpp

```diff
@@ -49,6 +49,8 @@ ZigbeeHandler::ZigbeeHandler(ZigbeeManager *zigbeeManager, QObject *parent) :
     registerObject<ZigbeeAdapter, ZigbeeAdapters>();
     registerEnum<ZigbeeNodeRelationship>();
     registerEnum<ZigbeeNodeRouteStatus>();
+//    registerEnum<ZigbeeClusterId>();
```

Comment:
Not required?

Reply:
No, I decided to go with the quint16 clusterId instead because otherwise the protocol would crap out on proprietary/unknown clusters.